x-pack/plugins/cases/public/components/custom_fields/toggle/create.tsx

```diff
@@ -15,13 +15,13 @@ const CreateComponent: CustomFieldType<CaseCustomFieldToggle>['Create'] = ({
   customFieldConfiguration,
   isLoading,
 }) => {
-  const { key, label } = customFieldConfiguration;
+  const { key, label, defaultValue } = customFieldConfiguration;
 
   return (
     <UseField
       path={`customFields.${key}`}
       component={ToggleField}
-      defaultValue={false}
+      config={{ defaultValue: defaultValue ? defaultValue : false }}
```

Comment:
The other way around, but yes!

[Reference here.](https://docs.elastic.dev/form-lib/core/default-value#order-of-precedence)

The initial value for the form(when it exists) is defined in the form config in `x-pack/plugins/cases/public/components/custom_fields/form.tsx`. `defaultValue={false}` had the highest preference so it would always overwrite the initial value. 

`useField` config has the lowest precedence so now it works fine. The form checks for the initial value and if it doesn't exist it checks for the default. 🙌 

This was a bit annoying to find 😅 and fwiw the tests take this into account. (I don't remember exactly which atm)